mnelab/model.py

```diff
@@ -145,6 +147,9 @@ def load(self, fname, *args, **kwargs):
                                      ftype=ftype, fsize=fsize, data=data,
                                      dtype=dtype))
 
+        if ext == ".hdr":
+            self.current["montage"] = True
```

Comment:
Thanks for the pointer @cbrnr. This works, but I still feel a cleaner solution may exist. I will revisit this as I get a better understanding of MNELAB.

Reply:
Yes, I was thinking that you could explicitly call `self.model.set_montage` with the correct `montage` object. The problem here seems to be that such a `montage` object doesn't exists as the channel locations are directly set in the loading function. Do you think it is possible to extract the montage part in `read_raw_nirx` into a separate step?

Reply:
To clarify, I mean calling `set_montage` inside `_load_nirx`.

Reply:
I think I will have to do some more reading of the MNE manuals. I think I have misunderstood what is meant by the term montage. The location of sensors (optodes) is stored in `info` in the `locs` array. I was assuming this is what was meant by a montage. I will look in to the MNE docs.

However, the code above enables the button in the MNELAB main window. If I click the button it displays the sensors in the correct position on a 2d head. So this is the outcome I desired. Now I just need to make the code correct :)

Reply:
I never seem to remember where MNE stores channel locations. According to the docs, it should be in `info["chs"]`, which is a list of dicts for each channel. Each dict has a `"loc"` key, which contains the coordinates of that channel. However, there's also `info["dig"]`, which is also a list of dicts for each channel. Each dict has an `"r"` key containing channel coordinates. Not sure what the difference between these two entries is, but I think the first one is used for montages.

When you do `raw.set_montage(...)`, MNE populates the `info["chs"]` list of dicts (specifically the `"loc"` values). But there's no way to determine the montage given the locations - which is why I'm saving the montage explicitly (partly also because MNELAB displays the montage name).

Reply:
According to [this tutorial](https://mne.tools/dev/auto_tutorials/intro/plot_40_sensor_locations.html), both `chs` and `dig` entries are populated. Maybe MNELAB should determine whether a montage is set (or in other words, if channel locations are available) by looking at these two keys. Then it would be unnecessary to store that extra information in `self.current["montage"]`, because this information would be available in `self.current["data"].info`...

Reply:
I've implemented this in #147. If you rebase, you can remove setting the montage explicitly, this should now be picked up automatically.

Reply:
Thanks. I appreciate your help here.

I will move on to adding the two required conversion steps next.